tests/linearizability/linearizability_test.go

```diff
@@ -212,20 +271,22 @@ type trafficConfig struct {
 	traffic     Traffic
 }
 
-func validateEventsMatch(t *testing.T, ops [][]watchEvent) {
-	// Move longest history to ops[0]
+func pickLongestHistory(ops [][]watchEvent) (longest []watchEvent, rest [][]watchEvent) {
```

Comment:
Just having 'sortDescByLenght' would have simpler contract and shorter implementation. 
Cost of sorting of O(3) pointers to lists is negligible. 

Now we have function that sounds like getter... but is mutating 'ops'. 

```
sort.Slice(ops, func(i, j []watchEvent) bool {
  return len(i)>len();
})
```